config/routes.rb

```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
   # Defines the root path route ("/")
   # root "articles#index"
+  resources :users, only: [:show, :index] 
+  resources :posts, only: [:show, :index]
```

Comment:
- [ ] Kindly implement the requirement asked by the previous reviewer, check the picture :arrow_down:  as an example
![image](https://user-images.githubusercontent.com/62705947/185413867-3f97219c-c509-47bc-a4ff-7a0d67b8c4fb.png)

Reply:
Thank you @GzimAsani